fileWithPRComments.js

```diff
@@ -0,0 +1,8 @@
+/* global QamineUtils */
+
+"use strict";
+
+(function($, window) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Move the invocation into the parens that contain the function. (wrap-iife)](https://app.codacy.com/manual/prodcodacytestuser/CheckPrCommentsPublicProjectProd/pullRequest?prid=5003319)